app/models/submission.rb

```diff
@@ -344,6 +346,16 @@ def can_view_plagiarism_report(type, user, session)
     )
   end
 
+  def user_can_peer_review?(user)
+    allowed = false
+    if assignment.peer_reviews && self.assigned_assessments
+      self.assigned_assessments.each do |review|
+        allowed = allowed || user.id == review.user_id
```

Comment:
>Perhaps you want the .any? method here instead of .each?
Yup! I was just writing it out long hand, didn't realize Ruby has a shorthand for it... thanks!